openlibrary/catalog/get_ia.py

```diff
@@ -154,8 +155,7 @@ def get_from_archive_bulk(locator):
 
     assert 0 < length < MAX_MARC_LENGTH
 
-    ureq = urllib.request.Request(url, None, {'Range': 'bytes=%d-%d' % (r0, r1)})
-    f = urlopen_keep_trying(ureq)
+    f = urlopen_keep_trying(url, {'Range': 'bytes=%d-%d' % (r0, r1)})
```

Comment:
```suggestion
    f = urlopen_keep_trying(url, params={'Range': 'bytes=%d-%d' % (r0, r1)})
```
Please make these calls as self-documenting as you can.

Reply:
ok.  I named it `headers`.